### PR TITLE
chore: upgrade to Husky 6

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

**Comment by nschonni** [comment]
Downgraded to 7 like the core repo because of the Node 10 requirement

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "eslint-plugin-jsdoc": "^30.5.1",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.1.4",
-    "husky": "^4.3.0",
+    "husky": "6.0.0",
```
Comment (alexander-akait) [inline package.json:28]: Please use `^` to get bug fixes and new features

Comment (nschonni) [reply package.json:28]: I'll update it, but there are no other releases to 6. I believe it was  a temporary one that works without the engine strict of Node 12 v7

Comment (nschonni) [reply package.json]: I've updated the range

**Comment by sokra** [comment]
Thanks

**Pull request merged**